### DISCUSSION TITLE
xml imports

### DISCUSSION THREAD

**Discussion by pfetrial** [FreeOpcUa/opcua-asyncio#385] [General]
Hello
I tried this morning to import a xml file into a server with import_xml and several errors during the import phase.

I made also another test : I took the customnodes.xml file ( stored in the repository of your project) and I tried to read it with SIOME ( the modeler from SIEMENS) . There was no error but no nodes where imported..

It seems to me that the import module is not compatible to the output of a "standard" uamodeler. Is this the case?
thx

**Comment by AndreasHeine** [comment]
can you provide the versions you use?
python:
asyncua:
SiOME:

can you provide the xml-file?

**Reply by AndreasHeine** [reply]
it should work
![Unbenannt](https://user-images.githubusercontent.com/56362817/103524647-2cd5ff80-4e7e-11eb-8a4b-0ab39e365f28.PNG)

if no nodes are created there might be a missing nodeset like DI or so? but therfor we need the stacktrace or the logger infos

**Comment by oroulet** [comment]
yes . that kind of things should work. it there are bugs. please report precise bug request for each of them.
